09:45:58.266834 INFO [otestpointd/application] commandline: otestpointd -d -l 3 -f /home/ubuntu/Desktop/emane-tutorial/23aliakim/persist/host/var/log/otestpointd1.log --pidfile /home/ubuntu/Desktop/emane-tutorial/23aliakim/persist/host/var/run/otestpointd1.pid --uuidfile /home/ubuntu/Desktop/emane-tutorial/23aliakim/persist/host/var/run/otestpointd1.uuid host-otestpointd1.xml 
09:45:58.266854 INFO [otestpointd/application] uuid: bfe071a8-86a6-42c4-b797-d45ceb8278bf
09:45:58.310937 INFO [otestpointd] discovery service listening on tcp://127.0.0.1:8881
09:45:59.658856 INFO [otestpoint-probe/node-1/0/manager] available probes: EMANE.PhysicalLayer.Counters.General EMANE.PhysicalLayer.Tables.Events EMANE.PhysicalLayer.Tables.Status
09:45:59.675289 INFO [otestpoint-probe/node-1/1/manager] available probes: EMANE.RFPipe.Counters.General EMANE.RFPipe.Tables.Events EMANE.RFPipe.Tables.Neighbor EMANE.RFPipe.Tables.Status
09:45:59.677616 ERROR [otestpointd/node-1/1/container] Traceback (most recent call last):

  File "/usr/lib/python2.7/dist-packages/otestpoint/emane/probebase.py", line 222, in start
    raise ProbeException("unable to determine build id")

ProbeException: unable to determine build id

09:55:46.638901 INFO [otestpointd/application] shutdown
